modules/nf-neuro/bundle/labelmap/meta.yml

```diff
@@ -43,10 +43,31 @@ args:
       type: string
       description: Select the colormap for colored trk (data_per_point).
       default: "jet"
-  - new_labelling:
+  - threshold:
+      type: float
+      description: Maximum distance between two points to be considered overlapping.
+      default: 0.001
+  - streamline_threshold:
+      type: float
+      description: Threshold for the minimum number of streamlines in a voxel to be included.
+      default: 2
+  - hyperplane:
       type: boolean
-      description: Use the new labelling method (multi-centroids).
+      description: Use hyperplane method (multi-centroids) instead of the euclidean method (single-centroid).
       default: false
+  - use_manhattan:
```

Comment:
Uniformize. Either `use_*` for both flags, or just the name. I don't believe we're uniform on that anywhere, so I let you decide which standard you want to apply here.